tests/test_models.py

```diff
@@ -29,3 +29,20 @@ def test_daily_mean_integers():
     # Need to use Numpy testing functions to compare arrays
     npt.assert_array_equal(daily_mean(test_input), test_result)
 
+@pytest.mark.parametrize('data, expected_standard_deviation', [
+    ([0, 0, 0], 0.0),
+    ([1.0, 1.0, 1.0], 0),
+    ([0.0, 2.0], 1.0)
+])
+def test_daily_standard_deviation(data, expected_standard_deviation):
+    from inflammation.models import s_dev
+    result_data = s_dev(data)['standard deviation']
+    npt.assert_approx_equal(result_data, expected_standard_deviation)
+
+
+def test_daily_standard_deviation_on_np_array():
+    from inflammation.models import s_dev
+    data = np.array([[0.0]])
+    expected_std_devs = [[0.0]]
+    out = s_dev(data)['standard deviation']
+    npt.assert_array_almost_equal(data, expected_std_devs)
```

Comment:
```suggestion
    npt.assert_array_almost_equal(out, expected_std_devs)
```
Here we are not testing the result of the test

Reply:
Indeed! Well caught.